### PR TITLE
Add responsibilities for local multiplayer game <documentation

### PR DESCRIPTION
**Pull request by bryanvanwijk**
I documented the responsiblities for the new classes for the local MultiPlayer feature.
Let me know if I missed something.

### CODE REVIEW

File: deliverables/sprint4/Exercise1.tex [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+In the table the responsibilities and collaborations are presented for every class.
+\begin{center}
+    \begin{tabular}{ | p{4.5cm} | p{3cm} | p{3cm} | p{3cm} | p{1cm} |}
+  \hline
+    Class & Responsibility & Collaborates with & Super & Sub \\ \hline
+   MultiPlayerGame & Players  & & Game & \\ \hline
+   MultiPlayerGameUIController & Key control & MultiPlayerGame  & GameUIController  & \\ \hline
+   MenuUIController& Conrol menu & MultiPlayerGameUI- Controller  &   &  \\ \hline
+   UiElementMultipleSpaceShips& draw multiple spaceShips & MultiPlayerGameUI- Controller  &  &  \\ \hline
+ MultipleSpaceShipsController& Controll multiple spaceships & SpaceShip  & SpaceShipController &  \\ \hline
```
Comment (doriandekoning) [inline deliverables/sprint4/Exercise1.tex]: I would change this to MultiSpaceShipsController to be consistent.

**Comment by doriandekoning** [comment]
Content looks good! But shouldn't the requirements be before the responsibilities?

**Comment by bryanvanwijk** [comment]
Yes, I think that would be better I changed it.

**Comment by doriandekoning** [comment]
Great!

**Comment by jochem725** [comment]
There are two spelling errors regarding "control".
If this is fixed, I agree with merging.

**Pull request merged**